src/dmd/astbase.d

```diff
@@ -109,7 +109,7 @@ struct ASTBase
         nodtor              = (1L << 24),   // don't run destructor
         nothrow_            = (1L << 25),   // never throws exceptions
         pure_               = (1L << 26),   // pure function
-        tls                 = (1L << 27),   // thread local
+        //                    (1L << 27),
```

Comment:
Remove the other declaration as well:

https://github.com/dlang/dmd/blob/fe263f5991ea571b0384c6facfe207a7b17efb44/src/dmd/declaration.d#L229